cli/src/cortx-prereqs.sh

```diff
@@ -251,16 +251,21 @@ fi
 
 echo -n "INFO: Checking if kernel version is correct.........................." 2>&1 | tee -a ${LOG_FILE}
 
+grep -q "Red Hat" /etc/*-release && { 
```

Comment:
does CentOS also return "Red Hat"?
and is it "Red Hat" or "RedHat"?

Reply:
1. On centos above grep command will return nothing and hence kernel version will be set to 3.10.0-1127.el7.x86_64
2. In /etc/os-release and /etc/redhat-release the release name  is mentioned as "Red Hat Enterprise Linux Server"